README.md

```diff
@@ -46,100 +50,116 @@ for the TaxCloud web service response in seconds. Default values:
+transaction.returned # returns "OK" or raises an error
+```
 
 ### Verifying Addresses
 
 [TaxCloud](http://www.taxcloud.com) optionally integrates with the USPS
 Address API. An address can be verified, which can also yield a 9-digit zip
 code that helps determine a more accurate tax rate.
 
-    address = TaxCloud::Address.new({
-      :address1 => '888 6th Ave',
-      :city => 'New York',
-      :state => 'New York',
-      :zip5 => '10001'
-    })
+```ruby
+address = TaxCloud::Address.new({
+  :address1 => '888 6th Ave',
+  :city => 'New York',
+  :state => 'New York',
+  :zip5 => '10001'
+})
 
-    verified_address = address.verify
-    verified_address.zip5 # 10001
-    verified_address.zip4 # 3502
-    verified_address.zip # 10001-3502
+verified_address = address.verify
+verified_address.zip5 # 10001
+verified_address.zip4 # 3502
+verified_address.zip # 10001-3502
+```
 
 ### Tax Codes
 [TaxCloud](http://www.taxcloud.com) maintains a list of all Taxability
 Information Codes or TICs, which can be found at https://taxcloud.net/tic.
 
 You can obtain all tax codes as well as lookup a tax code by ID.
 
-    TaxCloud::TaxCodes.all # a hash of all codes
-    tax_code = TaxCloud::TaxCodes[TaxCloud::TaxCodes::DIRECT_MAIL_RELATED]
-    tax_code.ticid # 11000 or TaxCloud::TaxCodes::DIRECT_MAIL_RELATED
-    tax_code.description # "Direct-mail related"
+```ruby
+TaxCloud::TaxCodes.all # a hash of all codes
+tax_code = TaxCloud::TaxCodes[TaxCloud::TaxCodes::DIRECT_MAIL_RELATED]
+tax_code.ticid # 11000 or TaxCloud::TaxCodes::DIRECT_MAIL_RELATED
+tax_code.description # "Direct-mail related"
+```
 
 Tax codes are organized in groups.
 
-    TaxCloud::TaxCode::Groups.all # a hash of all groups
-    tax_code_group = TaxCloud::TaxCode::Groups[TaxCloud::TaxCode::Groups::SCHOOL_RELATED_PRODUCTS]
-    tax_code_group.group_id # 3 or TaxCloud::TaxCode::Groups::SCHOOL_RELATED_PRODUCTS
-    tax_code_group.description # School Related Products
-    tax_code_group.tax_codes # a hash of all codes in this group
+```ruby
+TaxCloud::TaxCode::Groups.all # a hash of all groups
+tax_code_group = TaxCloud::TaxCode::Groups[TaxCloud::TaxCode::Groups::SCHOOL_RELATED_PRODUCTS]
+tax_code_group.group_id # 3 or TaxCloud::TaxCode::Groups::SCHOOL_RELATED_PRODUCTS
+tax_code_group.description # School Related Products
+tax_code_group.tax_codes # a hash of all codes in this group
+```
 
 Tax code constants are defined in `tax_code_constants.rb` and tax code group
 constants in `tax_code_group_constants.rb`. These files can be generated by
 running the following rake tasks.
 
-    TAXCLOUD_API_LOGIN_ID=... TAXCLOUD_API_KEY=... TAXCLOUD_USPS_USERNAME=... tax_cloud:tax_codes
-    TAXCLOUD_API_LOGIN_ID=... TAXCLOUD_API_KEY=... TAXCLOUD_USPS_USERNAME=... tax_cloud:tax_code_groups
+```ruby
+TAXCLOUD_API_LOGIN_ID=... TAXCLOUD_API_KEY=... TAXCLOUD_USPS_USERNAME=... tax_cloud:tax_codes
+TAXCLOUD_API_LOGIN_ID=... TAXCLOUD_API_KEY=... TAXCLOUD_USPS_USERNAME=... tax_cloud:tax_code_groups
+```
```

Comment:
These are rake/shell commands and shouldn't use ruby syntax highlighting.

Reply:
Whoops, you're totally right @drewtempelmeyer. Force-pushed. Thanks!